fpipy/raw.py

```diff
@@ -365,7 +364,8 @@ def demosaic(cfa, pattern, dm_method):
     -------
     xarray.DataArray
     """
-    pattern_name = BayerPattern.get(pattern).name
+    if type(pattern) is BayerPattern:
```

Comment:
`pattern` should always be passed through BayerPattern to validate any strings passed to demosaic (this is the main intent of using the BayerPattern Enum in the first place). If the `get` method is removed, the validation pass should be implemented in some other way.